quanta-t41/config.json

```diff
@@ -35,7 +35,6 @@
     },
     "workflowRoot": "workflows",
     "taskRoot": "tasks",
-    "httpTemplateRoot": "templates",
-    "httpStaticRoot": "static"
```

Comment:
OK, in the point of functionality integrity, your design makes sense. But I don't think your way is good enough. User can still place an empty `static` folder (without bmc & bios sub-folders) in skupack in order to make the on-http code happy.